server/src/delete-shot-button.js

```diff
@@ -75,16 +75,20 @@ exports.DeleteShotButton = class DeleteShotButton extends React.Component {
       deletePanelOpenClass = "active";
     }
 
+    const deleteButtonStyle = this.props.isIcon ? `button transparent trash ${deletePanelOpenClass}` :
+      `nav-button transparent icon-trash trash ${deletePanelOpenClass}`;
+    const deleteText = !this.props.isIcon ? <Localized id="shotPageDelete">
+        <span>Delete</span>
+      </Localized> : null;
```

Comment:
nit: Both of these ternaries use `isIcon` as their condition, so consider use an if-else instead.

Reply:
Done. Thanks!